zhaquirks/sinope/light.py

```diff
@@ -148,3 +149,67 @@ class SinopeDM2500ZB(SinopeTechnologieslight):
             }
         }
     }
+
+
+class SinopeDM2550ZB(SinopeTechnologieslight):
+    """DM2550ZB Dimmer."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=260 device_version=1
+        # input_clusters=[0, 2, 3, 4, 5, 6, 8, 1794, 2820, 2821, 65281]
+        # output_clusters=[3, 4, 10, 25]>
+        MODELS_INFO: [(SINOPE, "DM2550ZB")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha_p.PROFILE_ID,
+                DEVICE_TYPE: zha_p.DeviceType.DIMMER_SWITCH,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    DeviceTemperature.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    Metering.cluster_id,
+                    ElectricalMeasurement.cluster_id,
+                    Diagnostic.cluster_id,
+                    SINOPE_MANUFACTURER_CLUSTER_ID,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Time.cluster_id,
+                    Ota.cluster_id,
+                ],
+            }
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha_p.PROFILE_ID,
+                DEVICE_TYPE: zha_p.DeviceType.DIMMABLE_LIGHT,
```

Comment:
I assume this quirk is only necessary to make this device match the light platform in ZHA. @Adminiuga the original device type is: `zha_p.DeviceType.DIMMER_SWITCH` Should we add a device type entry to discovery for this to make it go to light by default?